### PR TITLE
Loadout patch

### PR DESCRIPTION
**Pull request by HeroesMaker**
I don't really have the time to check everything resently, sorry for that, please make sure everything is in working order before pushing it and tell me what need to be changed.

### CODE REVIEW

File: loadouts/factions/german.sqf [hunk 1]
```diff
@@ -132,19 +129,19 @@ _medicUNIFORM    = _commonUNIFORM;
...
 _reconUNIFORM    = _randomUNIFORM;
 
-_commonVEST = "BRM_IOTV_FLECK_RFL";
-_officerVEST = "BRM_IOTV_FLECK_REPAIR";
-_ftlVEST = "BRM_IOTV_FLECK_TL";
-_slVEST = "BRM_IOTV_FLECK_SL";
-_mgVEST = "BRM_IOTV_FLECK_SAW";
-_grenadierVEST = "BRM_IOTV_FLECK_GL";
-_medicVEST = "BRM_IOTV_FLECK_MED";
+_commonVEST = "BRM_Protec_"+_camo"_RFL";
```
Comment (Coryf88) [inline loadouts/factions/german.sqf:132]: `"BRM_Protec_"+_camo+"_RFL"`

File: loadouts/factions/german.sqf [hunk 2]
```diff
@@ -132,19 +129,19 @@ _medicUNIFORM    = _commonUNIFORM;
...
+_officerVEST = "BRM_Protec_"+_camo"_SL";
```
Comment (Coryf88) [inline loadouts/factions/german.sqf:133]: `"BRM_Protec_"+_camo+"_SL"`

File: loadouts/factions/german.sqf [hunk 3]
```diff
@@ -132,19 +129,19 @@ _medicUNIFORM    = _commonUNIFORM;
...
+_ftlVEST = "BRM_Protec_"+_camo"_TL";
```
Comment (Coryf88) [inline loadouts/factions/german.sqf:134]: `"BRM_Protec_"+_camo+"_TL"`

File: loadouts/factions/german.sqf [hunk 4]
```diff
@@ -132,19 +129,19 @@ _medicUNIFORM    = _commonUNIFORM;
...
+_slVEST = _officerVEST;
+_mgVEST = "BRM_Protec_"+_camo"_MG";
```
Comment (Coryf88) [inline loadouts/factions/german.sqf:136]: `"BRM_Protec_"+_camo+"_MG"`

File: loadouts/factions/german.sqf [hunk 5]
```diff
@@ -132,19 +129,19 @@ _medicUNIFORM    = _commonUNIFORM;
...
+_grenadierVEST = "BRM_Protec_"+_camo"_GL";
```
Comment (Coryf88) [inline loadouts/factions/german.sqf:137]: `"BRM_Protec_"+_camo+"_GL"`

File: loadouts/faction-list.sqf [hunk 6]
```diff
@@ -141,14 +141,10 @@ switch (toUpper(_faction)) do {
         #include "factions\frwinter.sqf"
     };
 
-    case "FRWOOD": {
+    case "FRANCE": {
         #include "factions\frwood.sqf"
     };
 
-    case "FRDESERT": {
```
Comment (Coryf88) [inline loadouts/faction-list.sqf:148]: Why is "FRWINTER" not removed?

**Review by Coryf88 — CHANGES_REQUESTED** [review]

File: loadouts/faction-list.sqf (hunk 6 above)
Comment (HeroesMaker) [reply loadouts/faction-list.sqf:148]: Different uniform type using TRYK mod, vanilla vest and RHS helmet, unless there is a way to completely change the loadout using the camo function it has to be in a different file.

**Review by Coryf88 — APPROVED** [review]

**Pull request merged**